python/hidet/graph/transforms/__init__.py

```diff
@@ -49,16 +49,16 @@ def optimize(graph: FlowGraph) -> FlowGraph:
     ret: FlowGraph
         The optimized flow graph.
     """
+    ctx = PassContext.current()
     passes = [
         subgraph_rewrite_pass(),
         automatic_mix_precision_pass(),
-        subgraph_rewrite_pass(),
+        subgraph_rewrite_pass(ctx.configs['quantize_patterns']),
```

Comment:
Consider to write a new pass named quantize which can use subgraph rewrite to implement the functionality, and make the passes list clean.